tests/func/test_status.py

```diff
@@ -61,27 +61,25 @@ def test_status_non_dvc_repo_import(tmp_dir, dvc, erepo_dir):
     }
 
 
-def test_status_before_and_after_dvc_init(tmp_dir, dvc, erepo_dir):
-    with erepo_dir.chdir():
-        erepo_dir.scm.repo.index.remove([".dvc"], r=True)
-        shutil.rmtree(".dvc")
-        erepo_dir.scm_gen("file", "first version")
-        erepo_dir.scm.add(["file"])
-        erepo_dir.scm.commit("first version")
-        old_rev = erepo_dir.scm.get_rev()
+def test_status_before_and_after_dvc_init(tmp_dir, dvc, external_git_dir):
+    with external_git_dir.chdir():
+        external_git_dir.scm_gen(
+            "file", "first version", commit="first version"
+        )
+        old_rev = external_git_dir.scm.get_rev()
 
-    dvc.imp(fspath(erepo_dir), "file", "file")
+    dvc.imp(fspath(external_git_dir), "file", "file")
 
     assert dvc.status(["file.dvc"]) == {}
 
-    with erepo_dir.chdir():
-        Repo.init()
-        erepo_dir.scm.repo.index.remove(["file"])
+    with external_git_dir.chdir():
+        external_git_dir.dvc = Repo.init()
+        external_git_dir.scm.repo.index.remove(["file"])
         os.remove("file")
-        erepo_dir.dvc_gen("file", "second version")
-        erepo_dir.scm.add([".dvc", "file.dvc"])
-        erepo_dir.scm.commit("version with dvc")
-        new_rev = erepo_dir.scm.get_rev()
+        external_git_dir.dvc_gen("file", "second version")
```

Comment:
Adding the `commit` kwarg is equivalent to `add` + `commit`.
```suggestion
        external_git_dir.dvc_gen("file", "second version", commit="version with dvc")
```